content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,61 @@ positional arguments:
 
 In order to track parameters and hyperparameters associated to machine learning
 experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. Parameters are defined using the the `-p`
-(`--params`) option of `dvc run`, using simple names like `epochs`,
+dependencies: _parameters_. They usually have simple names like `epochs`,
 `learning-rate`, `batch_size`, etc.
 
 In contrast to a regular <abbr>dependency</abbr>, a parameter is not a file (or
-directory). Instead, it consists of a _parameter name_ (or key) to find inside a
-YAML 1.2, JSON, TOML, or [Python](#examples-python-parameters-file) _parameters
-file_. Multiple parameter dependencies can be specified from one or more
-parameters files.
+directory). Instead, it consists of a _parameter name_ (or key) in a
+[parameters file](#parameters-file), where the _parameter value_ should be
+found. This allows you to define [stage](/doc/command-reference/run)
+dependencies more granularly: changes to other parts of the params file will not
+affect the stage. Parameters also prevent situations where several stages share
```

Comment:
Parameter dependencies ?

Reply:
Hmmm OK, changes.

But I'm not sure about the phrase "parameter dependency" in general. It would be simpler to only say "parameter" most of the time. I guess since we already use the full phrase everywhere in `params diff` and especially since there will be dvc.yaml parameters (`${param}`) now (which are not stage dependencies) maybe it's a good idea to keep the phrase in these refs.

Reply:
> especially since there will be dvc.yaml parameters (${param}) now

OK, I changed some instances to just "params" and other to "param deps" depending on that. See 7165723e.

Reply:
so, when we talk about `dvc.yaml` or `dvc.lock`, or how can put it there via `dvc run` it is parameter dependency. When we talk about a key/value in the `params.yaml` it is a parameter, that your code could read for example.

Reply:
Agreed! That's basically the context I was referring to 🙂 (except sometimes for 2.0 `dvc.yaml` its sometimes not dependency but that's another PR).